cockatrice/src/gamesmodel.cpp

```diff
@@ -66,15 +65,18 @@ QVariant GamesModel::data(const QModelIndex &index, int role) const
         case ROOM:
             return rooms.value(g.room_id());
         case CREATED: {
-            QDateTime then;
-            then.setTime_t(g.start_time());
-            int secs = then.secsTo(QDateTime::currentDateTime());
-
             switch (role) {
-                case Qt::DisplayRole:
+                case Qt::DisplayRole: {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 8, 0))
```

Comment:
while this is unrelated to @knitknit 's pull request, I think it's important to keep a bit of consistency